### PR TITLE
edited border points, deaths color ramp and uganda district names

### PR DESCRIPTION
**Pull request by lynnnakiganda**

### CODE REVIEW

File: data/region_border_points.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+var region_border_points = {
```
Comment (RonKbS) [inline data/region_border_points.js:1]: @lynnnakiganda , please remove `is03_1`, `iso3_2` and `Type` columns

Comment (lynnnakiganda) [reply data/region_border_points.js:1]: Okay

**Comment by lynnnakiganda** [comment]
I cannot use the same region.js file because it's missing 2 or 3 polygons for the districts in some countries that's why I need to update it with this one

**Comment by RonKbS** [comment]
> I cannot use the same region.js file because it's missing 2 or 3 polygons for the districts in some countries that's why I need to update it with this one

You can add those polygons to the file that's already in master...

**Pull request merged**